tests/test_object_model.py

```diff
@@ -857,3 +857,47 @@ def foo():
     assert wrapped.name == "foo"
     cache_info = next(ast_nodes[2].infer())
     assert isinstance(cache_info, astroid.Instance)
+
+
+def test_class_annotations() -> None:
+    """Test that the `__annotations__` attribute is avaiable in the class scope"""
+    annotations, klass_attribute = builder.extract_node(
+        """
+        class Test:
+            __annotations__  #@
+        Test.__annotations__  #@
+        """
+    )
+    # Test that `__annotations__` attribute is available in the class scope:
+    assert isinstance(annotations, nodes.Name)
+    # The `__annotations__` attribute is `Uninferable`:
+    assert next(annotations.infer()) is astroid.Uninferable
+
+    # Test that we can access the class annotations:
+    assert isinstance(klass_attribute, nodes.Attribute)
+
+
+def test_class_annotations_typed_dict() -> None:
+    """Test that we can access class annotations on various TypedDicts"""
```

Comment:
This is not important at the present moment, but elsewhere I've been encourage to avoid boilerplate like "test that" and "we", and I think it's a helpful edit ;)